services/null_action.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../translation'
+
+class NullAction
+  include ::Translation
+
+  def call(wait_time = 3)
+    sleep(wait_time)
+    translate(:nice_day)
```

Comment:
Try to use `at_exit`([docs](https://apidock.com/ruby/Kernel/at_exit)) to show goodbye message.